sdcm/nemesis.py

```diff
@@ -3613,17 +3613,11 @@ def remove_node():
             return res.exit_status
 
         # full cluster repair
-        up_normal_nodes = self.cluster.get_nodes_up_and_normal(verification_node)
         # Repairing will result in a best effort repair due to the terminated node,
         # and as a result requires ignoring repair errors
         with DbEventsFilter(db_event=DatabaseLogEvent.RUNTIME_ERROR,
                             line="failed to repair"):
-            for node in up_normal_nodes:
-                try:
-                    self.repair_nodetool_repair(node=node, publish_event=False)
-                except Exception as details:  # noqa: BLE001
-                    self.log.error(f"failed to execute repair command "
-                                   f"on node {node} due to the following error: {str(details)}")
+            self.target_node.run_nodetool(sub_cmd="cluster repair", publish_event=True)
```

Comment:
this works only with tablets, for vnodes based need to use manager repair

Reply:
looks like outdated comment, mark it as resolved